webpack/envs/productionConfig.js

```diff
@@ -21,6 +21,9 @@ exports.productionConfig = {
   devtool: "source-map",
   output: {
     filename: "[name].[contenthash].js",
+    publicPath: process.env.CDN_URL
+      ? `${process.env.CDN_URL}/assets/`
```

Comment:
I'm curious about how `/assets/` is included here. Isn't that part of the path whether or not an external host is used?

Reply:
The `productionConfig` inherits from our [baseConfig](https://github.com/artsy/force/blob/master/webpack/envs/baseConfig.js#L22), so this just prefixes that with the CDN, and then later in the asset helper we check to see if the cdn is present and ignore.